pysonde/sounding.py

```diff
@@ -400,6 +406,11 @@ def export(self, output_fmt, cfg):
         output = self.meta_data["launch_time_dt"].strftime(output)
         directory = os.path.dirname(output)
         Path(directory).mkdir(parents=True, exist_ok=True)
+
+        platform_name = cfg.main.get("platform").replace(" ", "_")
```

Comment:
Can this line be moved before l 398 so we do this step only once. `platform_name` can then be used in 398